src/styles/components/student-assignmentlist.styl

```diff
@@ -15,5 +15,11 @@
       margin-bottom: 1em
 
     .listitem > div:nth-child(1)
```

Comment:
How do you mean? In any case, these styles are not used anymore as the view is a table now. I should have removed them really.